tools/deploy.sh

```diff
@@ -1,6 +1,13 @@
 cp .env.example .env
-sed -i 's|AP_POSTGRES_PASSWORD=.*|AP_POSTGRES_PASSWORD='"$(openssl rand -hex 32)"'|g' .env
-sed -i 's|AP_JWT_SECRET=.*|AP_JWT_SECRET='"$(openssl rand -hex 32)"'|g' .env
-sed -i 's|ENCRYPTION_KEY=.*|ENCRYPTION_KEY='"$(openssl rand -hex 16)"'|g' .env
 
-echo "A .env file containing random passwords and secrets has been successfully generated."
\ No newline at end of file
+if [[ $OSTYPE == 'darwin'* ]]; then
```

Comment:
This is causing `run.sh: 3: [[: not found`

We can replace it with `if test "$OSTYPE" = "darwin"*; then` but can you please check if it works on your MacOS?

Reply:
@abuaboud thanks for your feedback! 

I tried your code and unfortunately it didn't work, so I googled again and found another solution: https://stackoverflow.com/a/17072017

Could you please check it on your side?

Reply:
This works with bash `bash tools/deploy.sh`

Reply:
Replacing the if with single equals make it work

`if [ "$(uname)" = "Darwin" ]; then`

Reply:
Tested it on mac with single quote, It works!

Merging!